src/components/GearColumnProvider.js

```diff
@@ -3,9 +3,9 @@ import Link from "./shared/Link"
 import {
   makeColumns,
   GENERATED_ID_COL_INDEX,
-  RESTRICTED_COL_INDEX,
   indexRender,
 } from "./shared/ColumnHelper"
+import { createRangeFilter } from "./shared/FilterHelper"
```

Comment:
I like having the filters in a separate helper. If you look at the fix-price-filtering branch right now, I don't have that there, so at some point i'll have to move it over, depends which PR is finished first I guess (I'm still having issues getting the reset to work properly).